apple/internal/resources.bzl

```diff
@@ -382,13 +402,15 @@ def _bucketize_typed(resources, bucket_type, *, owner = None, parent_dir_param =
     parent_dir_param when available.
 
     Args:
-        resources: List of resources to place in bucket_type.
         bucket_type: The AppleResourceInfo field under which to collect the resources.
         owner: An optional string that has a unique identifier to the target that should own the
             resources. If an owner should be passed, it's usually equal to `str(ctx.label)`.
         parent_dir_param: Either a string/None or a struct used to calculate the value of
             parent_dir for each resource. If it is a struct, it will be considered a partial
             context, and will be invoked with partial.call().
+        resources: List of resources to place in bucket_type or Dictionary of resources keyed by
```

Comment:
Odd this moved down.

Reply:
hrm yea seems like buildifier doesn't care about the ordering 🤔